### PR TITLE
Updated elasticsearch dependency

### PR DESCRIPTION
**Pull request by wachterjohannes**
This PR upgrades the elasticsearch library.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -29,7 +29,9 @@
         "behat/web-api-extension": "~1.0@dev",
         "behat/symfony2-extension": "~2.0@dev",
         "phpbench/phpbench": "~0.10.0",
-        "phpbench/tabular": "@dev"
+        "phpbench/tabular": "@dev",
+
+        "guzzlehttp/streams": "^3.0.0"
```
Comment (danrot) [inline composer.json]: Why is that needed? The changes in here don't seem to require guzzle.

Comment (wachterjohannes) [inline composer.json]: if i remove this line the `--prefer-lowest` test doesnt work anymore (because of composer) https://travis-ci.org/massiveart/MassiveSearchBundle/jobs/140464875#L182

Comment (wachterjohannes) [inline composer.json]: i havent found any other solution.

Comment (danrot) [inline composer.json]: Hm, looks like that's a bug in the elasticsearch library? But well, looks like we don't have any other choice at the moment.
